### PR TITLE
(chore): upgrade to RN 0.71

### PR DESCRIPTION
**Pull request by robinheinze**
# Description

[Trello Card](77-upgrade-to-react-native-071-sdk-48)

Summary of the changes and any helpful notes for reviewers and testers.

- Upgraded to RN 0.71 via Expo SDK 47.0.13 (as advised by Kudo from Expo in [this Twitter thread](https://twitter.com/kudochien/status/1613715668511363074))

# Checklist:

- [x] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features
- [x] I have run tests and linter

### CODE REVIEW

**Review by kateinkim — APPROVED** [review]
🚀

**Review by frankcalise — APPROVED** [review]
Looks good! Left a few comments with regards to the types and config file

File: package.json [hunk 1]
```diff
@@ -78,12 +78,13 @@
     "@react-native-community/cli-platform-ios": "^8.0.2",
     "@rnx-kit/metro-config": "^1.2.32",
     "@rnx-kit/metro-resolver-symlinks": "0.1.21",
+    "@tsconfig/react-native": "^2.0.2",
     "@types/i18n-js": "3.8.2",
-    "@types/jest": "26.0.19",
+    "@types/jest": "^29.2.1",
     "@types/react": "~18.0.24",
     "@types/react-native": "~0.70.6",
```
[the remark this answers is not in the document]
Comment (robinheinze) [reply package.json:85]: So Expo is lagging a bit and actually won't let me start the dev client without the types installed. I assume that will be fixed in 48

Screen Shot 2023-01-17 at 8 58 48 AM

File: .ruby-version [hunk 2]
```diff
@@ -0,0 +1 @@
+2.7.6
```
[the remark this answers is not in the document]
Comment (robinheinze) [reply .ruby-version:1]: It was from the diff helper, but now that I think about it, it's not super necessary without all the native stuff. I'll take it back out.

**Pull request merged**